# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/src/nsFontMetrics.h
# Commit: f8ca02252dfc
# Full Hash: f8ca02252dfc48d7342703e6a635d510b9dc0b5d
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-04-02 09:51:45
# Regressor Bug: 1619349
# File Overlap Count: 1
# Description:
#   Bug 1619349 - patch 1 - Record statistics about font-matching behavior in the presContext. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67787
# ==============================================================================

diff -r 01a2a527aa1a -r f8ca02252dfc gfx/src/nsFontMetrics.h
--- a/gfx/src/nsFontMetrics.h	Wed Apr 01 21:42:11 2020 +0000
+++ b/gfx/src/nsFontMetrics.h	Wed Apr 01 21:42:36 2020 +0000
@@ -24,6 +24,7 @@
 class nsDeviceContext;
 class nsAtom;
 struct nsBoundingMetrics;
+struct FontMatchingStats;
 
 namespace mozilla {
 namespace gfx {
@@ -61,6 +62,7 @@
     FontOrientation orientation = eHorizontal;
     gfxUserFontSet* userFontSet = nullptr;
     gfxTextPerfMetrics* textPerf = nullptr;
+    FontMatchingStats* fontStats = nullptr;
     gfxFontFeatureValueSet* featureValueLookup = nullptr;
   };
 